### DISCUSSION TITLE
User-Defined Variation Selectors

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#298] [Ideas]
Finally this rather obvious idea got an official status:
https://www.unicode.org/L2/L2024/24148-n5266-uvs_proposal.pdf

**Comment by psb1558** [comment]
Is the official status that it has been approved?

**Reply by jsbien** [reply]
No, I mean that it was written down and officially submitted. Actually  Peter Constable writes (http://blog.unicode.org/2024/08/highlights-from-unicode-technical.html)

> That proposed solution is problematic: existing stability policies and commitments prevent assigning more PUA code points and also prevent constraining existing PUA for certain uses. At the same time, there was opinion within UTC that the need expressed was reasonable, and there was openness to considering alternative solutions. One potential alternative that gained some interest was to establish a registration process, similar to what is defined in UTS #37 for ideographic variation sequences but intended for use with historic scripts.
